### PR TITLE
Version releases docs

### PR DESCRIPTION
**Pull request by PavelBulat**

### CODE REVIEW

File: docs/version_releases.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Version releases
```
Comment (ikorgik) [inline docs/version_releases.rst]: What do you think if we move that information to the Falcon Development section?

File: docs/version_releases.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+================
+
+Create new release
+------------------
+
+How create new release you can read `this <https://help.github.com/en/articles/creating-releases>`_.
```
Comment (ikorgik) [inline docs/version_releases.rst]: I would rephrase as "You can read official Github [instruction] for more info how to create releases."

File: docs/version_releases.rst [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+Requirements
+------------
+- New release should based on master branch.
```
Comment (ikorgik) [inline docs/version_releases.rst]: Missed "be": ... should _be_ based...

File: docs/version_releases.rst [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+- New release tag must have name something like 1.0.0.
```
Comment (ikorgik) [inline docs/version_releases.rst]: I would say: "New release tag should look like "1.0.0".

File: docs/version_releases.rst [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+A normal version number MUST take the form X.Y.Z where X, Y, and Z are non-negative integers, and MUST NOT contain leading zeroes.
```
Comment (ikorgik) [inline docs/version_releases.rst]: It doesn't explain what major/minor and patch versions mean, I would rather just copy/paste a text from summary section on semver.org (https://semver.org/#summary).
Also I would include a link to semver site for more information.

**Review by ikorgik — CHANGES_REQUESTED** [review]

File: docs/version_releases.rst (hunk 1 above)
Comment (ikorgik) [reply docs/version_releases.rst]: Done.

**Review by ikorgik — APPROVED** [review]

File: docs/version_releases.rst (hunk 2 above)
Comment (PavelBulat) [reply docs/version_releases.rst]: Done.

File: docs/version_releases.rst (hunk 3 above)
Comment (PavelBulat) [reply docs/version_releases.rst]: Done.

File: docs/version_releases.rst (hunk 4 above)
Comment (PavelBulat) [reply docs/version_releases.rst]: Changed.

File: docs/version_releases.rst (hunk 5 above)
Comment (PavelBulat) [reply docs/version_releases.rst]: Changed.
